pyautomagic/preprocessing/performEOGRegression.py

```diff
@@ -1,46 +1,42 @@
 import numpy as np
 import logging
+import mne
 
-
-def performEOGRegression(eeg, eog, *args):
+def performEOGRegression(raw):
+  
     """Performs linear regression to remove EOG artifact from the EEG data
 
         Parameters
         ----------
-        eeg: np.ndarray
-            EEG signal with the EOG artifacts
-        eog: np.ndarray
-            EOG signal
-        *args
-            variable length argument list
+        raw: MNE raw data structure
 
         Returns
         -------
-        clean_eeg: np.ndarray
-                   Cleaned EEG signal from EEG artifacts
+        clean_EEG: np.ndarray
+                   Cleaned EEG signal from EOG artifacts
 
         """
-    # checks if EOG Regression should be skipped or not depending on the function arguments
-    if len(args[0]) == 0:
+
+    EEG=raw.get_data()
+    EOG_idx=mne.pick_types(raw.info, eog=True)
+    EOG=EEG[EOG_idx]
+    # checks if EOG Regression should be skipped or not depending
+    # on whether EOG was recorded
+    if EOG_idx.shape[0] == 0:
         logging.warning('EOG regression skipped')
```

Comment:
Here, you can perhaps add why it was skipped. E.g. a message saying it was skipped "because no EOG data in the mne <filename>."